gulpfile.js

```diff
@@ -34,7 +34,7 @@ gulp.task('js', () => {
 })
 
 gulp.task('docs', function() {
-  return run(`cd .. && ${DOCS_COMMAND}`).exec()
+  return run(DOCS_COMMAND).exec()
```

Comment:
I had to change this, otherwise it tried to run the DOCS_COMMAND in the parent directory. Let me know if I need to revert this for some reason.